eox_tenant/backends/persistent.py

```diff
@@ -0,0 +1,34 @@
+"""
+Site configuration backend that reads the configuration from the database
+"""
+
+
+class CompatibleDatabaseSiteConfigBackend(object):
```

Comment:
Is this a completely new backend unlinked from microsite backends?

Reply:
Yes, but for now this backend reads/loads configurations. The second phase of the development will handle more operations.